src/main/webapp/WEB-INF/tags/instructor/results/remindButton.tag

```diff
@@ -0,0 +1,13 @@
+<%@ tag description="instructorFeedbackResults - remind button" %>
+<%@ tag import="teammates.common.util.Const" %>
+
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<%@ attribute name="remindButton" type="teammates.ui.template.InstructorFeedbackResultsRemindButton" required="true" %>
+
+<div style="display: inline-block; padding-right: 5px;" class="remind-no-response">
+    <a href="#" data-actionlink="${remindButton.urlLink}" class="${remindButton.className}" data-toggle="modal"
+            <c:if test="${remindButton.disabled}">disabled="disabled"</c:if> data-target="#remindModal">
```

Comment:
Not sure myself. If both are equally justified, we can update the standard to match the more common usage. But if one option has a stronger justification, we'll have to switch to that one.